hack/e2e/run-rp-and-e2e.sh

```diff
@@ -65,9 +65,31 @@ register_sub() {
 }
 
 run_e2e() {
-    RESOURCEGROUP=$ARO_RESOURCEGROUP make test-e2e
+    export E2E_ENTRYPOINT="docker run --privileged \
+                --network host \
+                --env-file <(env | egrep 'AZURE|E2E|ARO') \
+                --env LOCATION \
+                --env CI \
+                --env RESOURCEGROUP=$ARO_RESOURCEGROUP \
+                --env CLUSTER=$CLUSTER \
+                --env RP_MODE \
+                --entrypoint e2e.test $ARO_IMAGE -test.timeout 180m -test.v -ginkgo.v"
+    RESOURCEGROUP=$ARO_RESOURCEGROUP E2E_ENTRYPOINT=$E2E_ENTRYPOINT make test-e2e
 }
 
+get_e2e_image_version(){
+    curl -s -w "\n%{http_code}" "https://${ARO_VERSION_SA}.blob.core.windows.net/rpversion/${LOCATION}" | {
+    read body
+    read code
+    if [[ $code == "200" ]]; then
+        echo $body
+    else
+        echo "latest"
```

Comment:
I can't think when this latest logic would be necessary - can we simplify right down to `curl -s "https://${ARO_VERSION_SA}.blob.core.windows.net/rpversion/${LOCATION}"` ?

Reply:
not quite. You get this from blob if somebody somewhere sets logic wrong:
```
 curl -s "https://blobname.blob.core.windows.net/rpversion/test"
<?xml version="1.0" encoding="utf-8"?><Error><Code>BlobNotFound</Code><Message>The specified blob does not exist.
RequestId:9d911e86-801e-000a-7b3a-b7c848000000
Time:2020-11-10T08:25:03.9554945Z</Message></Error>
```
:/ There might be other way but I don't know from top of my head to ignore outputs sometimes.